src/styles/auto-prefix.js

```diff
@@ -28,7 +28,7 @@ export default {
         userAgent: userAgent,
       });
 
-      return prefixer.prefix;
+      return prefixer;
```

Comment:
> return (style) => prefixer.prefix(style);

That makes sense to me.

> Now, what about the set() method? It will be removed but the components that use this method are not having their styles transformed right now.

I think the easiest way to deal with this once an for all is to migrate the components that are using `set()` to use the `prefixAll` method now included in muiTheme.